ui/readme-terra-ui-packages.md

```diff
@@ -0,0 +1,29 @@
+# Using Packages from @terra-ui-packages
+
+Terra UI publishes a [set of packages](https://github.com/DataBiosphere/terra-ui/tree/dev/packages) on every commit. These are stored in a public registry for use by anyone. We can and do use packages from this registry in our codebase.
+
+## Using Packages
+
+The [packages](https://github.com/DataBiosphere/terra-ui/tree/dev/packages) directory in the [Terra UI GitHub repo](https://github.com/DataBiosphere/terra-ui) contains a set of directories each with its own package.json file. Here is a current (2023-08-31) listing:
+```
+$ grep -R \"name\": packages
```

Comment:
What is this grep statement doing?

Reply:
Looking for the literal string: `"name":` to match that field in the JSON file.

Reply:
Ahh ok, this command (when run in the terra-ui repo) will give you the list of the package.json files. Thanks for clarifying.

Reply:
I added a little bit to the command try to clarify that this is being run within the Terra UI repo directory. Hopefully that will help future readers.